### DISCUSSION TITLE
Invalid User/Org

### DISCUSSION THREAD

**Discussion by diamant3** [community/community#61535] [Mobile] [answered]
Everything i search username or org and then when you click one of them are showing like this, i don't know why.
I tried to re login and delete data/cache but no luck

![Screenshot_20230723_072440.jpg](https://github.com/community/community/assets/71203851/a57dfc15-72dd-4bbf-9897-f29dd9ebfaca)

**Comment by GhostOf0days** [comment]
Please provide more details for reproducibility. It works on my end.

**Reply by diamant3** [reply]
here's my issue demo:

https://github.com/community/community/assets/71203851/500641e0-1451-4c61-954d-21fcc1610dd0

**Reply by GhostOf0days** [reply]
It does not have that issue for me. Please create a [Github Support ticket](https://support.github.com/). Also please provide them the exact Github Mobile Version (check mobile app settings), mobile OS version, device, and the video you sent here. They will be able to escalate the issue.

**Comment by aconsuegra** [comment, marked as answer]
Hi @diamant3 , thanks for reaching out with your feedback. This is a known issue on our Android app that has already been fixed in our Beta build, and will be available in Production from tomorrow. If you are blocked by it, I recommend you to join our Beta testers program in Google Play. You can leave the testers program later once the fix has been fully rolled out to production.

**Reply by diamant3** [reply]
Got it, Thank you!
